airbyte-webapp/src/components/base/DropDown/SelectContainer.tsx

```diff
@@ -3,7 +3,14 @@ import { components, ContainerProps } from "react-select";
 
 import { OptionType } from "./DropDown";
 
-export const SelectContainer: React.FC<ContainerProps<OptionType, false>> = (props) => {
+export const SelectContainer: React.FC<
+  ContainerProps<OptionType, false> & {
+    selectProps: {
+      "data-testid": string;
```

Comment:
Using an `aria-label` would be more accessible than a `data-testid` and should still be findable by React Testing Library and Cypress.

Reply:
https://github.com/airbytehq/airbyte/blob/6b5e4fb6e6975d833f7c9054fc67e81ec1d53f8f/airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.test.tsx#L323
We're using this utility that doesn't offer an `aria-label` option. Changing this breaks the tests. Suggestions?

Reply:
We could query by role and use `{name: ${ariaLabelValue}}` probably.  However, this whole form appears to utilize `data-testid` widely so sounds like a bigger project for another time!